### PR TITLE
Dockerfile: option --text doesn't exist

### PR DESCRIPTION
**Pull request by Jadaw1n**

### CODE REVIEW

**Comment by bmw** [comment]
Thanks for the PR @Jadaw1n. This pointed out to us a problem with command line parsing. `--text` is a valid option, but it's not being handled correctly when it is the only option specified on the command line. Because of this, I reopened #442 which should resolve the issue.

In the mean time, the way to write the command line to get the desired behaviour is `letsencrypt --text run`. If you'd like to modify your PR to reflect this, I'll happily merge. Regardless, thanks for pointing out the problem to us!

File: Dockerfile [hunk 1]
```diff
@@ -62,5 +62,5 @@ RUN virtualenv --no-site-packages -p python2 /opt/letsencrypt/venv && \
 # bash" and investigate, apply patches, etc.
 
 ENV PATH /opt/letsencrypt/venv/bin:$PATH
-# TODO: is --text really necessary?
```
Comment (kuba) [inline Dockerfile]: That comment was meant to make sure that `dialog` (ncurses) display works in Docker out of the box. I think I might have had some problems, back when I left the comment. Unfortunately, Docker is broken on my sid box, so I can't verify it. 

Did you actually manage to run the ncurses interface without any problems?

Comment (bmw) [inline Dockerfile]: That's a better solution if it works reliably. Tested on Arch and `dialog` worked fine in Docker.

Comment (kuba) [inline Dockerfile]: In this case, I'm happy to have this PR merged as is (despite author's original intention). :)

**Comment by bmw** [comment]
lgtm

**Comment by Jadaw1n** [comment]
Would be nice if you'd document the switch somehow... It's not in the help page (--help), so I thought the feature got removed...

**Comment by bmw** [comment]
No problem! This made the `Dockerfile` usable again and allows people to make their own choice between a ncurses or text interface.

`--text` is documented in `--help`, but `--help` just lists the subcommands and tells you how you can get more help about a specific topic.
```
letsencrypt --help    

  letsencrypt [SUBCOMMAND] [options] [domains]

The Let's Encrypt agent can obtain and install HTTPS/TLS/SSL certificates.  By
default, it will attempt to use a webserver both for obtaining and installing
the cert.  Major SUBCOMMANDS are:

  (default) everything Obtain & install a cert in your current webserver
  auth                 Authenticate & obtain cert, but do not install it
  install              Install a previously obtained cert in a server
  revoke               Revoke a previously obtained certificate
  rollback             Rollback server configuration changes made during install
  config_changes       Show changes made to server config during installation

Choice of server for authentication/installation:

  --apache          Use the Apache plugin for authentication & installation
  --nginx           Use the Nginx plugin for authentication & installation
  --standalone      Run a standalone HTTPS server (for authentication only)
  OR:
  --authenticator standalone --installer nginx

More detailed help:

  -h, --help [topic]    print this message, or detailed help on a topic;
                        the available topics are:

   all, apache, automation, nginx, paths, security, testing, or any of the
   subcommands

```
If you choose `all` as the topic, you'll see:
```
letsencrypt --help all
usage: 
  letsencrypt [SUBCOMMAND] [options] [domains]

The Let's Encrypt agent can obtain and install HTTPS/TLS/SSL certificates.  By
default, it will attempt to use a webserver both for obtaining and installing
the cert.  

positional arguments:
  SUBCOMMAND
    run                 Obtain a certificate and install.
    auth                Authenticate & obtain cert, but do not install it.
    install             Install a previously obtained cert in a server.
    revoke              Revoke a previously obtained certificate.
    rollback            Rollback server configuration changes made during
                        install.
    config_changes      Show changes made to server config during installation
    plugins             List server software plugins.

optional arguments:
  -h, --help            show this help message and exit
  -c CONFIG_FILE, --config CONFIG_FILE
                        config file path (default: None)
  -v, --verbose         This flag can be used multiple times to incrementally
                        increase the verbosity of output, e.g. -vvv. (default:
                        -3)
  -t, --text            Use the text output instead of the curses UI.
                        (default: False)
  -m EMAIL, --email EMAIL
                        Email used for registration and recovery contact.
                        (default: None)
  -d DOMAIN, --domains DOMAIN
  --duplicate           Allow getting a certificate that duplicates an
                        existing one (default: False)
```
Where `--text` can be found under optional arguments. I removed the output about other topics for brevity. Regardless, sorry for the confusion!

**Comment by Jadaw1n** [comment]
Ah, cool thanks!

**Pull request merged**